### PR TITLE
Fix/insufficient fund create nation protobuf fixes

### PR DESCRIPTION
**Pull request by borjantrajanoski**
Important fixes related to the ```v``` and ```chainID``` parameter of the ETH tx need to be merged in relation to the protocol buffers

### CODE REVIEW

**Comment by hunght** [comment]
Look good to me

**Comment by borjantrajanoski** [comment]
@Mnuzz I pushed a few changes which remove the pre-compiled binaries and pave the road for Bitrise generated builds

File: src/services/upstream/upstream.js [hunk 1]
```diff
@@ -120,7 +119,7 @@ export default class UpstreamService {
     const valueEth = valueBn.div(new BigNumber(10).pow(18)).toString(10);
     const transaction = {
       to,
-      data: normalizeHexValue(data),
```
Comment (seland) [inline src/services/upstream/upstream.js]: @borjantrajanoski Is there any reason to remove `normalizeHexValue` call? I think it should be there.

Comment (borjantrajanoski) [reply src/services/upstream/upstream.js]: @seland It was removed on purpose as ```normalizeHexValue``` was being called one more time [here](https://github.com/Bit-Nation/BITNATION-Pangea-mobile/blob/d23fbb55049f174de69aad7c3297e83c88789e88/src/services/ethereum/CustomSigner.js#L113)

This was causing the data to be prefixed by an additional `0x` which was causing the data to get malformed.

**Comment by Mnuzz** [comment]
Will merge this as soon as I can verify a build with panthalassa

**Pull request merged**